sys/errors.go

```diff
@@ -0,0 +1,8 @@
+package sys
```

Comment:
Please remove this file. ErrNotImplemented is not used, and ErrInvalidAction will not be used if you remove other unused code (see below).